index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body class="layout">
+  <header class="header">
+    <nav>
+      <img src="images/burgers.svg" alt="Menu">
+    </nav>
+  </header>
+  <main>
```

Comment:
- [optional] You might want to add a logo to your design by using either your name or image. This is important for Navigation and User Experience: Logos often serve as a clickable link to the homepage, acting as a navigational tool for users. By clicking on the logo, visitors can easily return to the main page from anywhere on the website. This improves the user experience and provides a convenient way to explore the site.